docs/resources.rst

```diff
@@ -36,6 +36,9 @@ Solidity Integrations
     * `Superblocks Lab <https://lab.superblocks.com/>`_
         Browser-based IDE. Built-in browser-based VM and Metamask integration (one click deployment to Testnet/Mainnet).
 
+    * `Solidity IDE <https://github.com/System-Glitch/Solidity-IDE>`_
```

Comment:
This list should be sorted alphabetically, so please move it above Superblocks - actually could you also sort `Solium` into its place? :)

Reply:
#120787a